sonar-project.properties

```diff
@@ -0,0 +1,9 @@
+sonar.host.url = https://sonarcloud.io
+sonar.organization = getyoti
+sonar.projectKey = getyoti:java
+sonar.projectName = Yoti Java SDK
```

Comment:
I don't mind which one we go with, but would be good to be consistent with the .NET project name. That is currently `dotnet-sdk`, I would be inclined to not have "Yoti" in the name, as `getyoti` or `Yoti Ltd` seems to mostly be present whenever we're looking at it:
![image](https://user-images.githubusercontent.com/1381991/72827011-e342d380-3c71-11ea-9eea-c7228374a7e5.png)
Having said that, it also looks nicer to have ".NET SDK" and "Java SDK" rather that "dotnet-sdk" and "java-sdk", so we could go with that, and I'll update .NET too?

Reply:
Yeah makes sense - I'll update it to "Java SDK"

Reply:
Fixed in ab3013b